packages/v4/patternfly-docs/content/design-guidelines/components/expandable-section/expandable-section.md

```diff
@@ -39,3 +41,7 @@ Use an expandable section with a blue border to add more visual weight. This var
 ### Expandable section in a documentation page
 
 <img src="./img/documentation.png" alt="Expandable section in a documentation page" width="644"/>
+
+
+
+For more information regarding accessibility, visit the [expandable-section accessibility tab](https://www.patternfly.org/v4/components/expandable-section/accessibility).
```

Comment:
just noticed you put a dash between "expandable" and "section" in this link, is there a reason for that? I feel it's probably not needed since everyone else it's referred to there isn't one